web-server/app/api/stream/route.ts

```diff
@@ -0,0 +1,170 @@
+import { exec } from 'child_process';
+import { createReadStream, FSWatcher, watch } from 'fs';
+
+import { handleRequest, handleSyncServerRequest } from '@/api-helpers/axios';
+import { ServiceNames } from '@/constants/service';
+import {
+  ServiceStatus,
+  UPDATE_INTERVAL,
+  LogFile,
+  LOG_FILES
+} from '@/constants/stream';
+
+const execPromise = (command: string): Promise<string> => {
+  return new Promise((resolve, reject) => {
+    exec(command, (error, stdout) => {
+      if (error) {
+        return reject(error);
+      }
+      resolve(stdout);
+    });
+  });
+};
+
+const checkServiceStatus = async (
+  serviceName: ServiceNames
+): Promise<boolean> => {
+  try {
+    switch (serviceName) {
+      case ServiceNames.API_SERVER:
+        const apiResponse = await handleRequest('');
+        return apiResponse.message.includes('hello world');
```

Comment:
is this intentional, or accidentally pushed testing code?

Reply:
Okay so this is part of the logic that checks if the backend server is up or not we have an end point in both the sync server and API server that returns this response `hello world` I am thinking of adding a special route like /health that can do the same job what you think  @samad-yar-khan?